lib/redshift_ecto/connection.ex

```diff
@@ -515,7 +515,7 @@ if Code.ensure_loaded?(Postgrex) do
     defp sources_unaliased(prefix, sources, pos, limit) when pos < limit do
       current =
         case elem(sources, pos) do
-          {table, schema} ->
+          {table, schema, _alias} ->
```

Comment:
Ecto 3 allows naming for bindings. I believe that's what the third argument is, but it'll need to be double checked.